### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by VallyPepyako**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,4 +48,4 @@ Update task will upload the extension to your space on Contentful.
 
 Copyright &copy; Contentful Developer Relations
 
-Licensed under the [MIT license](https://github.com/contentful/developer-relations/ui-country-select/blob/master/LICENSE).
\ No newline at end of file
+Licensed under the [MIT license](https://github.com/contentful/developer-relations/ui-country-select/blob/master/LICENSE).
```
Comment (stefanjudis) [inline README.md]: Can you update this path, too? 😊

**Review by stefanjudis — CHANGES_REQUESTED** [review]
@KikahaPlus44 👋🏻

Thanks so much for opening this PR. Copy and paste is evil. :)

As you cleaned up paths I have a tiny additional change request. :)

File: README.md (hunk 1 above)
Comment (VallyPepyako) [reply README.md]: :) why not

**Comment by stefanjudis** [comment]
Thanks so much. 👍

**Pull request merged**